battleship.py

```diff
@@ -80,7 +114,15 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
+    p=0
+    for i in range(3):
+        a1=ship[i]
+        if grid[a1[0]][a1[1]]==1:
+            p=p+1
+    if p==3:
+        return True
+    else:
+        return False
 

```

Comment:
Good Job!! Please try to improve the readability of the code by changing the 3 to ship.